modules/gdnative/include/android/godot_android.h

```diff
@@ -46,6 +48,9 @@ extern "C" {
 
 JNIEnv *GDAPI godot_android_get_env();
 jobject GDAPI godot_android_get_activity();
+jobject GDAPI godot_android_get_surface();
+EGLDisplay GDAPI godot_android_get_display();
+EGLContext GDAPI godot_android_get_context();
```

Comment:
@BastiaanOlij I'm guessing `godot_android_get_display` and `godot_android_get_context` are for the Oculus mobile SDK?
If that's the case, then these methods are not needed as we can generate EGLDisplay and EGLContext on the native side. The newer versions of the Oculus mobile SDK actually require it. 
See [commit #5c3cf3f3ff4f09c62d35b02375cb30cb0c1da36c](https://github.com/m4gr3d/godot_oculus_mobile/commit/5c3cf3f3ff4f09c62d35b02375cb30cb0c1da36c) for a reference implementation.

Reply:
Ah that would be great. We started gearvr module on the basis of the native activity example. Originally godot was a native application only to be changed to SurfaceView later on but most of the old code was still in the code base confusing the heck out of us. I cleaned that up a little while ago.

I'm guessing we should see how much we can change over to follow the SurfaceView example in the SDK, it's only some of the setup thats different. Ah indeed by the looks of your commit you've already nearly completed that :thumbsup:

Reply:
On that subject, I might merge in https://github.com/GodotVR/godot_oculus_mobile/pull/20 tonight as I did a whole bunch of cleanup and improvements so we may be doubling up on stuff